osu.Desktop/DiscordRichPresence.cs

```diff
@@ -139,9 +182,25 @@ private void updateStatus()
             client.SetPresence(presence);
         }
 
+        private void onJoin(object sender, JoinMessage args)
+        {
+            game.Window?.Raise(); // users will expect to be brought back to osu! when joining a lobby from discord
```

Comment:
Probably don't want this level of commentary.
```suggestion
            game.Window?.Raise();
```